taskana-adapter-camunda-system-connector/src/test/java/pro/taskana/camunda/camundasystemconnector/acceptance/RetrieveCamundaTaskAccTest.java

```diff
@@ -48,10 +54,24 @@ public void setUp() {
     }
 
     @Test
-    public void testGetActiveCamundaTasks() {
+    public void testGetActiveCamundaTasks() throws ParseException {
 
         String timeStamp = "2019-01-14T15:22:30.811+0000";
 
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+                .withLocale(Locale.ROOT)
+                .withZone(ZoneId.of("UTC"));
+
+        Instant createdAfter = Instant.from(dateTimeFormatter.parse("2019-01-14T15:22:29.811+0000"));
+
+
+        Date date = java.sql.Timestamp.valueOf(createdAfter.atZone(ZoneId.systemDefault()).toLocalDateTime());
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+
+        String expectedBody = "{\"createdAfter\": \"" + formatter.format(date) + "\"}";
```

Comment:
expectedBody is not used in the class - delete the line

Reply:
changed